baselines/naming/protos/google/naming/v1beta1/naming.proto.baseline

```diff
@@ -112,21 +112,6 @@ service Naming {
       body: "*"
     };
   }
-
```

Comment:
FYI: the lines removed from the test proto correspond to a feature (reserved words) that I'm not going to cover soon.